packages/browser/src/node/index.ts

```diff
@@ -81,6 +81,13 @@ export default (project: WorkspaceProject, base = '/'): Plugin[] => {
           },
         }
       },
+      transform(code, id) {
+        if (id.includes('loupe/loupe.js')) {
```

Comment:
We can just bundle `loupe` to be honest, but then we should probably bundle `chai`, but I don't see that happening 🤷🏻

Reply:
Just in case, can you clarify the issues with bundling `chai`?
I guess the obvious one is that it affects vitest's package size. Another one I thought about is that a user code relying on `import "chai"` would end up with different instance of `chai` from what Vitest sees internally.
Do you have anything else in mind?